src/appshell/internal/startupscenario.cpp

```diff
@@ -169,9 +169,8 @@ mu::Uri StartupScenario::startupPageUri(StartupModeType modeType) const
     case StartupModeType::StartWithNewScore:
         return HOME_URI;
     case StartupModeType::StartWithScore:
-        return NOTATION_URI;
     case StartupModeType::ContinueLastSession:
```

Comment:
Make sure you don't break the recovery of the previous session if the app was crashed. Because we also use this flag in this case
see `StartupScenario::run`
```
    if (isMainInstance && sessionsManager()->hasProjectsForRestore()) {
        modeType = StartupModeType::ContinueLastSession;
    }
```